coa-helm/templates/cromwell.yaml

```diff
@@ -78,6 +78,7 @@ data:
     }
 
     filesystems {
+      drs.global.config.martha.url = "{{ .Values.config.marthaUrl }}"
```

Comment:
Think this needs to change to `.Values.config.drsUrl` now too :)

Reply:
Whoops, thanks!